internal/armada/scheduling/lease.go

```diff
@@ -38,20 +38,39 @@ func LeaseJobs(
 	onJobLease func([]*api.Job),
 	request *api.LeaseRequest,
 	activeClusterReports map[string]*api.ClusterUsageReport,
+	activeClusterAllocatedJobs []*api.Job,
 	clusterPriorities map[string]map[string]float64,
 	activeQueues []*api.Queue,
 ) ([]*api.Job, error) {
 	resourcesToSchedule := common.ComputeResources(request.Resources).AsFloat()
 	currentClusterReport, ok := activeClusterReports[request.ClusterId]
 
+	totalCapacity := &common.ComputeResources{}
+	for _, clusterReport := range activeClusterReports {
+		totalCapacity.Add(clusterReport.ClusterAvailableCapacity)
+	}
+
+	resourceAllocatedByQueue := map[string]common.ComputeResources{}
+	for _, job := range activeClusterAllocatedJobs {
```

Comment:
Could we use `currentClusterReport.Queues` here instead of loading all leased jobs?